memcached-image/memcached-image.xsl

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+<xsl:output method="xml" encoding="UTF-8" indent="yes"/>
+<xsl:template match="/">
+<xsl:comment> OBS-ExclusiveArch: aarch64 x86_64 </xsl:comment>
+<xsl:comment>
+  This file is generated,
+  don't manually modify this file,
+  please check README
+</xsl:comment>
+
+<image schemaversion="6.9" name="memcached-image" xmlns:suse_label_helper="com.suse.label_helper">
+
+  <description type="system">
+    <author>Thomas Bechtold</author>
+    <contact>tbechtold@suse.com</contact>
+    <specification>memcached container</specification>
+  </description>
+
+  <preferences>
+    <type image="docker">
+      <xsl:attribute name="derived_from">
+        <xsl:value-of select="param/container_base_image" />
+      </xsl:attribute>
+      <containerconfig
+          user="memcached"
+          additionaltags="%LONG_VERSION%,%LONG_VERSION%-%RELEASE%"
+          maintainer="SUSE Cloud Team &lt;cloud@suse.com&gt;">
+          <xsl:attribute name="name">
+            <xsl:value-of select="param/container_name" />
+          </xsl:attribute>
+          <xsl:attribute name="tag">
+            <xsl:value-of select="param/container_tag" />
+          </xsl:attribute>
+
+        <expose>
+          <port number="11211/tcp"/>
+          <port number="11211/udp"/>
+        </expose>
+        <subcommand execute="/usr/sbin/memcached"/>
+        <entrypoint execute="/usr/local/bin/docker-entrypoint.sh"/>
```

Comment:
why did we pick local/bin rather than local/sbin (the memcached is in sbin/) ?

Reply:
Switched to `/usr/local/sbin/`